src/screens/explore/BlendedAbout/index.tsx

```diff
@@ -43,8 +42,7 @@ const BlendedAbout = ({ route, navigation }: BlendedAboutProps) => {
       }));
     }
     if (course.trainer.picture?.link) setTrainerPictureSource({ uri: course.trainer.picture.link });
```

Comment:
J'ai une erreur ici `undefined is not an object (evaluating 'course.trainer.picture')`